include/openenclave/host.h

```diff
@@ -122,6 +123,17 @@ typedef struct _oe_enclave_setting_context_switchless
     size_t max_enclave_workers;
 } oe_enclave_setting_context_switchless_t;
 
+
+/**
+ * The setting for config_id/config_svn on Ice Lake platform.
+ */
+typedef struct _oe_enclave_setting_config_data
+{
+    uint8_t enclave_config_id[64];
+    uint16_t enclave_config_svn;
+    bool is_config_id_setting_mandatory;
```

Comment:
I would invert the definition of this flag so the default is that it fails if the configuration is provided but is not supported at runtime by the current TEE e.g. `bool ignore_if_unsupported`